src/jsdoc.ts

```diff
@@ -111,19 +141,10 @@ const JSDOC_TAGS_WHITELIST = new Set([
  * these will cause Closure Compiler issues and should not be used.
  */
 const JSDOC_TAGS_BLACKLIST = new Set([
-  'constructor',
-  'enum',
-  'extends',
-  'implements',
-  'interface',
-  'lends',
-  'private',
-  'public',
-  'record',
-  'template',
-  'this',
-  'type',
-  'typedef',
+  'augments', 'class',     'constructs', 'constructor', 'enhance',    'enhanceable',
+  'enum',     'extends',   'field',      'function',    'implements', 'interface',
+  'lends',    'namespace', 'private',    'public',      'record',     'static',
+  'template', 'this',      'type',       'typedef',
```

Comment:
clang-format decided on this. I tend to agree, but actually if you have a ws ignoring differ, it's fine.